src/components/Messages/Message/index.js

```diff
@@ -141,5 +142,6 @@ export const Message = memo(({
 			})}
 		</MessageContent>
 		{!compact && !message.t && <MessageTime normal={!me} inverse={me} ts={ts} />}
+		{message.t === 'jitsi_call_started' && message.callStatus === 'accept' ? <ShowCallTime stime={ts} /> : null}
```

Comment:
@Deepak-learner Can we reuse the existing system messages instead of creating a new Component for the same functionality? You can refer to [this PR](https://github.com/RocketChat/Rocket.Chat.Livechat/pull/554) for more info